extension-manifest-v3/src/background/stats.js

```diff
@@ -173,4 +243,16 @@ if (__PLATFORM__ !== 'safari' && __PLATFORM__ !== 'firefox') {
       urls: ['<all_urls>'],
     },
   );
+
+  chrome.webRequest.onErrorOccurred.addListener(
+    (details) => {
+      if (details.error !== 'net::ERR_BLOCKED_BY_CLIENT') return;
```

Comment:
It looks like there is a way to check if the request was blocked on the client side - so it must be our extension... or another one :P This is the only drawback, which I could imagine - if a user has more adblockers installed, it might be false positive - but still, the request is blocked, so the information is correct.